text0:
  text_content: "N"
  position_x: 0.9
  position_y: 0.1
  scale: 2.0
  thickness: 3
  red: 0
  green: 255
  blue: 0
  alpha: 0.8
text1:
  text_content: "W"
text2:
  text_content: "S"
text3:
  text_content: "E"
compass0:
  topic: "imu"
  timeout: 0.5
  scale: 15.0
  thickness: 3
  position_x: 0.9
  position_y: 0.1
